src/core/document/schema.pl

```diff
@@ -376,6 +388,7 @@
 refute_schema(Validation_Object,Witness) :-
     database_prefixes(Validation_Object, Prefixes),
     is_simple_class(Validation_Object,Class),
+    \+ is_json_class(Validation_Object, Class),
```

Comment:
So if I understand correctly, as long as something is tagged as a json class, we just believe that?
Isn't there some more elaborate check to be done to make sure this json object doesn't contain weirdness that should not be part of it?

Reply:
It's possible that you could confuse the schema by direct triple load, but I don't think it's possible by loading documents.